recipes/libudev-cos6-x86_64/COPYING

```diff
@@ -0,0 +1,339 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
THats because udev apparently is still under GPL....
```
LICENSE:
        LGPLv2.1+ for all code
        - except src/basic/MurmurHash2.c which is Public Domain
        - except src/basic/siphash24.c which is CC0 Public Domain
        - except src/journal/lookup3.c which is Public Domain
        - except src/udev/* which is (currently still) GPLv2, GPLv2+
```

https://cgit.freedesktop.org/systemd/systemd/tree/README

Reply:
Its strange, apparently libudev is LGPL, but `udev` is GPL.
https://cgit.freedesktop.org/systemd/systemd/tree/src/libudev/libudev.h

Not too sure.

Reply:
Yes, I'm pretty sure `libudev` is LGPL. It's just weird that the RPM packaged the GPL file. My best guess is that's just a packaging mistake in the RPM.

Reply:
I can't really find the centos7 package for libudev, can you? maybe we can use that to double check which is the actual license to include?

Reply:
It's in `systemd` now, AFAIR.

Reply:
Your right, on centos:latest, `libudev-devel` resolves to `systemd-devel`

Reply:
So should pull the LGPL license from the source tree?

```
/*
 * libudev - interface to udev device information
 *
 * Copyright (C) 2008-2009 Kay Sievers <kay.sievers@vrfy.org>
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 */
```

Reply:
looking through ther source rpm, they include the source tar ball. I think the udev developers simply forgot to include a copy of the license themselves. Later they make it clear and add a file LICENSE.LESSER. I'll just get a copy of LGPLv2